.github/workflows/CI.yml

```diff
@@ -18,7 +12,6 @@ jobs:
           - '1' # Leave this line unchanged. '1' will automatically expand to the latest stable 1.x release of Julia.
           #- 'nightly'
         os:
-          - macOS-latest
```

Comment:
Is there a specific reason why removed macOS from the CI? It would be great if this package is compatible with macOS...

Reply:
The main issue for the CI is that I need to install BART into the environment and on MacOS the steps are not straightforward.